app/models/radio.rb

```diff
@@ -0,0 +1,11 @@
+class Radio < ActiveRecord::Base
+  has_many :user_radios
+  has_many :users, through: :user_radios
+
+  validates :name,
+  uniqueness: {scope: :category},
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.